addons/medical_gui/initSettings.sqf

```diff
@@ -51,3 +51,183 @@
     [[0, 1, 2], [ELSTRING(common,Disabled), ELSTRING(Medical_Treatment,Anyone), ELSTRING(Medical_Treatment,Medics)], 1],
     false
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(bloodLossColor_0), 
+    "COLOR",
+    [LSTRING(BloodLossColor_0_DisplayName), LSTRING(BloodLossColor_Description)],
+    [ELSTRING(medical,Category), LSTRING(BloodLossColors)],
+    [1.00, 1.00, 1.00, 1],
+    false // isGlobal
+] call CBA_fnc_addSetting;
```

Comment:
I think it would be easier to maintain with for-loops
`for "_index" from 0 to 9 do {`
and a single stringtable entry
`<English>Blood Loss Color %</English>`

if this sounds ok I am happy to do the change

Reply:
Do you want to call `CBA_fnc_addSettings` in for loop? There is different default color for each entry.